boards/arm/nucleo_f091rc/doc/nucleof091rc.rst

```diff
@@ -142,27 +151,106 @@ This interface is supported by the openocd version included in the Zephyr SDK.
 Flashing an application to Nucleo F091RC
 ----------------------------------------
 
-Here is an example for the :ref:`blinky-sample` application.
+Here is an example for the :ref:`hello_world` application.
+
+First, run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the Nucleo F091RC board
+can be found. For example, under Linux, :code:`/dev/ttyACM0`.
+
+Then build and flash the application in the usual way.
 
 .. zephyr-app-commands::
-   :zephyr-app: samples/basic/blinky
+   :zephyr-app: samples/hello_world
    :board: nucleo_f091rc
    :goals: build flash
 
-You will see the LED blinking every second.
-
 Debugging
 =========
 
-You can debug an application in the usual way.  Here is an example for the
-:ref:`hello_world` application.
+You can debug an application in the usual way. Here is an example for the
+:ref:`button-sample` application.
 
 .. zephyr-app-commands::
-   :zephyr-app: samples/hello_world
+   :zephyr-app: samples/basic/button
    :board: nucleo_f091rc
    :maybe-skip-config:
    :goals: debug
 
+Testing the Nucleo board
+************************
+
+Testing the button and the LED
+==============================
+
+There are 2 samples that allow you to test that the button and LED on
+the board are working properly with Zephyr:
```

Comment:
I'd delete the code block below and just say here:

     ... with Zephyr:  :ref:`blinky-sample` and :ref:`button-sample`

Reply:
will change